README.md

```diff
@@ -79,15 +79,29 @@ player.
 #### New Feature
 * Plan-domizer
   * Ability to create a custom seed by placing items where you want them.
-  * Spoiler log is now in JSON format
-  * Plan-domizer uses the spoiler log JSON format
+  * Spoiler log is now in JSON format.
+  * Plan-domizer uses the spoiler log JSON format.
 * Cow Sanity
   * Playing Epona's Song for a cow for the first time gives an item.
 * Starting Age Option
   * Can now start as child, adult, or random.
+* Entrance Randomizer
+  * Ability to randomize dungeon entrances or all interior entrances.
+  * Gossip stone added to Lake Hylia to change water level after beating Morpha to keep dungeon entrance accessible with Gold Scales.
```

Comment:
It seems everyone is still confused about this, the new gossip stone in Lake Hylia which allows you to freely change the water level after beating Morpha is not related to Entrance Shuffle whatsoever. It's there in all settings and fully part of the randomizer now.
Also it's probably a good idea to mention the "Child May Drain Lake Hylia" option in the changelog, which is also not directly related to Entrance Shuffle btw.

Reply:
Will update, thanks for the info :)

Reply:
No worries, the rest seems good to me tho. 👍

Reply:
Always useful getting extra pairs of eyes on these changes <3